src/buffer/out/textBuffer.cpp

```diff
@@ -1995,25 +1995,10 @@ size_t TextBuffer::SpanLength(const til::point coordStart, const til::point coor
 // - end - where to end getting text
 // Return Value:
 // - Just the text.
-std::wstring TextBuffer::GetPlainText(const til::point& start, const til::point& end) const
+std::wstring TextBuffer::GetPlainText(const til::point start, const til::point end) const
 {
-    std::wstring text;
-    auto spanLength = SpanLength(start, end);
-    text.reserve(spanLength);
-
-    auto it = GetCellDataAt(start);
-
-    for (; it && spanLength > 0; ++it, --spanLength)
-    {
-        const auto& cell = *it;
-        if (cell.DbcsAttr() != DbcsAttribute::Trailing)
-        {
-            const auto chars = cell.Chars();
-            text.append(chars);
-        }
-    }
-
-    return text;
+    const auto req = CopyRequest::FromConfig(*this, start, end, true, false, false, false);

```

Comment:
FWIW I believe we should remove the constructor from `CopyRequest` and instead sanitize it in each individual function that accepts it. This would allow the callers to use designated initializers like so:
```cpp
CopyRequest req{
    .beg = ...,
    .end = ...,
    .singleLine = true,
};
```
This would turn `CopyRequest` into a true request struct IMO, similar to how you sanitize web requests on the server only.